XamlAnimatedGif.Shared/Animator.cs

```diff
@@ -64,7 +64,7 @@ private Animator(Stream sourceStream, Uri sourceUri, GifDataStream metadata, Rep
             _timingManager = CreateTimingManager(metadata, repeatBehavior);
         }
 
-        internal static async Task<Animator> CreateAsync(Image image, Uri sourceUri, RepeatBehavior repeatBehavior = default(RepeatBehavior))
+        public static async Task<Animator> CreateAsync(Image image, Uri sourceUri, RepeatBehavior repeatBehavior = default(RepeatBehavior))
```

Comment:
Agree, it could use its own issue. I rolled back this commit.

I needed the hability to show animated GIFs in IimageBrushes.
To achieve this, it would probably need more than one helper method. What about stopping the animation when the brush is dismissed ?